pkg/batch/cmd/server/server.go

```diff
@@ -560,6 +531,62 @@ func (s *server) createMySQLClient(
 	)
 }
 
+func (s *server) startMultiPubSub(
+	ctx context.Context,
+	processors *processor.Processors,
+	logger *zap.Logger,
+) *subscriber.MultiSubscriber {
+	bytes, err := os.ReadFile(*s.subscriberConfig)
+	if err != nil {
+		logger.Fatal("subscriber: failed to read subscriber config",
+			zap.Error(err),
+		)
+		return nil
+	}
+	var configMap map[string]subscriber.Configuration
+	if err := json.Unmarshal(bytes, &configMap); err != nil {
+		logger.Fatal("subscriber: failed to unmarshal subscriber config",
+			zap.Error(err),
+		)
+		return nil
+	}
+	multiSubscriber := subscriber.NewMultiSubscriber(
+		subscriber.WithLogger(logger),
+	)
+	for name, config := range configMap {
+		p, err := processors.GetProcessorByName(name)
+		if err != nil {
+			logger.Fatal("subscriber: processor not found",
+				zap.String("name", name),
+				zap.Error(err),
+			)
+			return nil
+		}
+		multiSubscriber.AddSubscriber(subscriber.NewSubscriber(
+			name, config, p,
+			subscriber.WithLogger(logger),
+		))
+	}
+	multiSubscriber.Start(ctx)
+	return multiSubscriber
+}
+
+func (s *server) registerProcessorMap(
+	ctx context.Context,
```

Comment:
We can remove the context since it's being used.

Reply:
Done.